### PR TITLE
merge Addnewevent to main

### PR DESCRIPTION
**Pull request by JPCatze**
add default color , fix some bug

### CODE REVIEW

File: lib/src/features/addevent/addevent.dart
Comment (INZDY) [inline lib/src/features/addevent/addevent.dart:1]: Remove print

**Comment by INZDY** [comment]
@JPCatze Please merge with main and resolve conflicts in firestore.dart before pushing.
You can refer to that conflict below.

File: lib/src/features/addevent/addevent.dart
Comment (JPCatze) [reply lib/src/features/addevent/addevent.dart:1]: removed & pushed

**Review by INZDY — APPROVED** [review]
Clear to merge

**Pull request merged**